huaweicloud/services/dds/data_source_huaweicloud_dds_instances.go

```diff
@@ -0,0 +1,309 @@
+				Computed:    true,
+				Description: `Indicates the ID of the instance.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the DB instance name.`,
+			},
+			"vpc_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the VPC ID`,
+			},
+			"subnet_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the VPC ID.`,
+			},
+			"security_group_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the security group ID of the DDS instance.`,
+			},
+			"disk_encryption_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the disk encryption ID of the instance.`,
+			},
+			"mode": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Specifies the mode of the database instance.`,
+			},
+			"db_username": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the DB Administator name.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the the DB instance status.`,
+			},
+			"enterprise_project_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the enterprise project id of the dds instance.`,
+			},
+			"nodes": {
+				Type:        schema.TypeList,
+				Elem:        BackupInstanceNodeSchema(),
+				Computed:    true,
+				Description: `Indicates the instance nodes information.`,
+			},
+			"tags": common.TagsSchema(),
+		},
+	}
+	return &sc
+}
+
+func BackupInstanceNodeSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the node ID.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the node name.`,
+			},
+			"role": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the node role.`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the node type.`,
+			},
+			"private_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the private IP address of a node.`,
+			},
+			"public_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the EIP that has been bound on a node.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the node status.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func resourceBackupRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+
+	var mErr *multierror.Error
```

Comment:
remove this line to line:223

Reply:
the code is automatically generated, the style is unified